### PR TITLE
[test](field) Basic usage of testing the Field

### PR DESCRIPTION
**Pull request by ZI-MA**
# Proposed changes

do a simple unit test for field

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/test/vec/core/field_uint64_test.cpp [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    EXPECT_EQ(Int32(114514), a.get_value());
+    EXPECT_EQ(UInt32(3), a.get_scale());
+    EXPECT_EQ(a, b);
+    EXPECT_LE(a, aa);
+}
+
+/*
+需要补位，所以会导致溢出
+非法case，需要保证scale相同才能比较大小关系
+*/
```
Comment (yiguolei) [inline be/test/vec/core/field_uint64_test.cpp:120]: no chinese comment

File: be/test/vec/core/field_uint64_test.cpp [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+// Decimal64
+TEST(FieldTest, Test_Field_Decimal64) {
+    constexpr Int64 val1 = 114514e12, val2 = 554514e12;
```
Comment (yiguolei) [inline be/test/vec/core/field_uint64_test.cpp:123]: check the string value of decimal<type> and datetype

**Comment by ZI-MA** [comment]
run buildall

**Comment by ZI-MA** [comment]
run p1

**Comment by ZI-MA** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.34 seconds
 stream load tsv:          445 seconds loaded 74807831229 Bytes, about 160 MB/s
 stream load json:         22 seconds loaded 2358488459 Bytes, about 102 MB/s
 stream load orc:          59 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          79.2 seconds inserted 10000000 Rows, about 126K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230523111335_clickbench_pr_148959.html

**Comment by ZI-MA** [comment]
run p0

**Comment by ZI-MA** [comment]
run be

**Comment by ZI-MA** [comment]
run beut

**Pull request closed without merging**